ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/DistributingExecutorTest.java

```diff
@@ -30,6 +30,7 @@
 import io.confluent.ksql.KsqlExecutionContext;
 import io.confluent.ksql.exception.KsqlTopicAuthorizationException;
 import io.confluent.ksql.metastore.MetaStore;
+import io.confluent.ksql.parser.KsqlParser.ParsedStatement;
```

Comment:
Can you add a specific test case to this file that ensures that the write protocol is followed? It should ensure that first we start the transaction, then verify, then write out the statement, and then commit.

Reply:
added one